xbmc/utils/URIUtils.cpp

```diff
@@ -968,8 +968,7 @@ bool URIUtils::IsInternetStream(const CURL& url, bool bStrictCheck /* = false */
 
   // Special case these
   if (url.IsProtocol("ftp") || url.IsProtocol("ftps")  ||
-      url.IsProtocol("dav") || url.IsProtocol("davs")  ||
-      url.IsProtocol("sftp"))
+      url.IsProtocol("dav") || url.IsProtocol("davs"))
```

Comment:
why would you remove this?

Reply:
If this needs to be checked the addon has to be asked, but don't assume anything in Kodi.

Reply:
well i won't merge this before you do the necessary code rewrites.

Reply:
@notspiff I could add a todo note and leave code like it is

Reply:
yes. it's not like this was done for rtmp for instance..

Reply:
updated